imports/ui/components/Card.jsx

```diff
@@ -110,22 +103,8 @@ class CardComponent extends Component {
               </div>
             )}
           </div>
-          {this.props.currentUser && (
-            <div>
-              <IconButton
-                style={{ display: this.state.isFav ? "none" : "" }}
-                onClick={this.onFavPressed}
-              >
-                <AddIcon />
-              </IconButton>
-              <IconButton
-                style={{ display: this.state.isFav ? "" : "none" }}
-                onClick={this.onFavPressed}
-              >
-                <CheckedIcon />
-              </IconButton>
-            </div>
-          )}
+          {/* Favorite icon is a part of the stretch goal to add wishlist, use later */}
```

Comment:
I think this comment should be removed in the refactor PR that I did. Maybe during merge conflict didn't pick the incoming change?

Reply:
Removed.